synctl

```diff
@@ -2352,7 +2373,71 @@ class UpdateSyntheticTest(SyntheticTest):
                                       verify=self.insecure)
 
         if update_result.status_code == 200:
-            print(f"{alert_id} updated")
+            print(f"alert {alert_id} updated")
+        elif update_result.status_code == 400:
+            print(f'Error: {update_result}', update_result.content)
+        elif update_result.status_code == 429:
+            self.exit_synctl(-1, "Too Many Requests")
+        else:
+            print(
+                f'update alert {alert_id} failed, status code: {update_result.status_code}, {update_result.text}')
+
+    def enable_smart_alert(self, alert_id):
+        """API https://instana.github.io/openapi/#operation/enableSyntheticAlertConfig"""
+        self.check_host_and_token(self.auth["host"], self.auth["token"])
+        host = self.auth["host"]
+        token = self.auth["token"]
+
+        if alert_id is None:
+            print("alert id should not be none")
+            sys.exit(1)
+        else:
+            put_url = f"{host}/api/events/settings/global-alert-configs/synthetics/{alert_id}/enable"
+
+        headers = {
+            'Content-Type': 'application/json',
+            "Authorization": f"apiToken {token}"
+        }
+
+        update_result = requests.put(put_url,
+                                     headers=headers,
+                                     timeout=60,
+                                     verify=self.insecure)
+
+        if update_result.status_code == 204:
+            print(f"alert {alert_id} enabled")
+        elif update_result.status_code == 400:
+            print(f'Error: {update_result}', update_result.content)
+        elif update_result.status_code == 429:
+            self.exit_synctl(-1, "Too Many Requests")
+        else:
+            print(
+                f'update alert {alert_id} failed, status code: {update_result.status_code}, {update_result.text}')
+
+    def disable_smart_alert(self, alert_id):
```

Comment:
Merge these two fuctions enable_smart_alert and disable_smart_alert into a single function, there is only micro difference.